maven build fails with Java 11

## Orb version
`v0.7.3`

## What happened:

CircleCI executor machine tag: ubuntu-2004:202101-01 
Build using Java 11 fails with the following error:

```
[ERROR] Unknown lifecycle phase "mx2048m". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy. -> [Help 1]
org.apache.maven.lifecycle.LifecyclePhaseNotFoundException: Unknown lifecycle phase "mx2048m". You must specify a valid lifecycle phase or a goal in the format <plugin-prefix>:<goal> or <plugin-group-id>:<plugin-artifact-id>[:<plugin-version>]:<goal>. Available lifecycle phases are: validate, initialize, generate-sources, process-sources, generate-resources, process-resources, compile, process-classes, generate-test-sources, process-test-sources, generate-test-resources, process-test-resources, test-compile, process-test-classes, test, prepare-package, package, pre-integration-test, integration-test, post-integration-test, verify, install, deploy, pre-clean, clean, post-clean, pre-site, site, post-site, site-deploy.
```

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ Usage example:
           mvn_path: ./mvnw
           steps:
             - cloudesire/maven_install_library:
-                repo_url: git@github.com:ClouDesire/abstract-connector.git
+                repo_url: git@github.com:my-organization/my-library.git
             - run:
                 name: Build
                 command: ./mvnw -B package
@@ -45,4 +45,3 @@ Ex:
 ```
 
 For the full list of parameters and default values check the [command page](https://circleci.com/developer/orbs/orb/cloudesire/common#commands-docker_run_and_test) in the orb documentation.
-

--- a/README.md
+++ b/README.md
@@ -7,6 +7,23 @@ CircleCI Orb developed by the ClouDesire Team.
 
 ## Commands and examples
 
+### Maven: install external library
+
+To install an external library from another repository, use the `maven_install_library` command. It downloads the repository and installs the library on the build VM/container. 
+Usage example:
+
+```yaml
+- cloudesire/maven_with_cache:
+          mvn_path: ./mvnw
+          steps:
+            - cloudesire/maven_install_library:
+                repo_url: git@github.com:ClouDesire/abstract-connector.git
+            - run:
+                name: Build
+                command: ./mvnw -B package
+```
+
+
 ### Docker: run and test container
 
 To run and check if a container starts correctly, you can use the `docker_run_and_test` command. It runs the container and check, by default, the url http://localhost:8080/actuator/health waiting for the container to be `UP`.

--- a/src/commands/maven_install_library.yml
+++ b/src/commands/maven_install_library.yml
@@ -5,7 +5,7 @@ parameters:
   maven_command:
     type: string
     default: "clean install"
-  maven_opts:
+  maven_cmd_opts:
     type: string
     default: "-Dmaven.test.skip=true -Dmaven.javadoc.skip=true"
   maven_path:
@@ -28,7 +28,7 @@ steps:
       name: Install maven library from << parameters.repo_url >>
       environment:
         MAVEN_COMMAND: << parameters.maven_command >>
-        MAVEN_OPTS: << parameters.maven_opts >>
+        MAVEN_CMD_OPTS: << parameters.maven_cmd_opts >>
         MAVEN_PATH: << parameters.maven_path >>
         REPO_DIR: << parameters.repo_dir >>
         REPO_URL: << parameters.repo_url >>

--- a/src/scripts/maven_install_library.sh
+++ b/src/scripts/maven_install_library.sh
@@ -3,6 +3,10 @@ basename=$(basename $REPO_URL)
 REPO_NAME=${basename%.*}
 
 echo "Installing library '${REPO_NAME}' from folder ${REPO_DIR}/${REPO_NAME}"
+echo "Maven path: ${MAVEN_PATH}"
+echo "Maven command: ${MAVEN_COMMAND}"
+echo "Maven options: ${MAVEN_OPTS}
+
 cd "${REPO_DIR}/${REPO_NAME}"
 ${MAVEN_PATH} ${MAVEN_COMMAND} ${MAVEN_OPTS}
 

--- a/src/scripts/maven_install_library.sh
+++ b/src/scripts/maven_install_library.sh
@@ -8,5 +8,5 @@ echo "Maven command: ${MAVEN_COMMAND}"
 echo "Maven options: ${MAVEN_OPTS}"
 
 cd "${REPO_DIR}/${REPO_NAME}"
-${MAVEN_PATH} ${MAVEN_COMMAND} ${MAVEN_OPTS}
+${MAVEN_PATH} ${MAVEN_COMMAND} "${MAVEN_OPTS}"
 

--- a/src/scripts/maven_install_library.sh
+++ b/src/scripts/maven_install_library.sh
@@ -5,7 +5,7 @@ REPO_NAME=${basename%.*}
 echo "Installing library '${REPO_NAME}' from folder ${REPO_DIR}/${REPO_NAME}"
 echo "Maven path: ${MAVEN_PATH}"
 echo "Maven command: ${MAVEN_COMMAND}"
-echo "Maven options: ${MAVEN_OPTS}
+echo "Maven options: ${MAVEN_OPTS}"
 
 cd "${REPO_DIR}/${REPO_NAME}"
 ${MAVEN_PATH} ${MAVEN_COMMAND} ${MAVEN_OPTS}

--- a/src/scripts/maven_install_library.sh
+++ b/src/scripts/maven_install_library.sh
@@ -5,8 +5,8 @@ REPO_NAME=${basename%.*}
 echo "Installing library '${REPO_NAME}' from folder ${REPO_DIR}/${REPO_NAME}"
 echo "Maven path: ${MAVEN_PATH}"
 echo "Maven command: ${MAVEN_COMMAND}"
-echo "Maven options: ${MAVEN_OPTS}"
+echo "Maven options: ${MAVEN_CMD_OPTS}"
 
 cd "${REPO_DIR}/${REPO_NAME}"
-${MAVEN_PATH} ${MAVEN_COMMAND} "${MAVEN_OPTS}"
+${MAVEN_PATH} ${MAVEN_COMMAND} "${MAVEN_CMD_OPTS}"
 

